tests/app/dao/test_organisation_dao.py

```diff
@@ -260,7 +261,7 @@ def test_dao_get_invited_organisation_user(sample_invited_org_user):
     assert invited_org_user == sample_invited_org_user
 
 
-def test_dao_get_invited_organisation_user_returns_none(notify_db):
+def test_dao_get_invited_organisation_user_returns_none():
```

Comment:
I don't think the fixture is necessary because no database manipulation should occur.